THIRD-PARTY-NOTICES.TXT

```diff
@@ -9,6 +9,8 @@ bring it to our attention. Post an issue or email us:
 The attached notices are provided for information only.
 
 
+TODO: Should we keep this notice?
```

Comment:
@joperezr @RussKie what do you think folks? There's references to OTel left at all.

Reply:
Good catch. No, it's no longer required if we don't have OTel code.

Reply:
I'll remove it then

Reply:
Done